### PR TITLE
Feat(PBOT-96): project link query method

### PR DESCRIPTION
**Pull request by colbyallenc**
ProjectLink method will use a provided `MySQLDriver` instance to execute the MySQL Query via Knex.
SQLDriver uses global instance in bot.js

### CODE REVIEW

File: lib/actions/projectOverviewResponse.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ class ProjectOverviewResponse {
   }
 
   sendToBot(bot, message) {
-    const listLinks = this.projectLinks.map((projectLink, i) => `*  *${projectLink.label}* ${projectLink.url}`).join('\n');
+    const listLinks = this.projectLinks.map(projectLink => `*  *${projectLink.label}* ${projectLink.url}`).join('\n');
```
Comment (nrambeck) [inline lib/actions/projectOverviewResponse.js]: I'm guessing this was a linting error that you fixed. If this comes up in the future, I recommend handling unrelated fixes in a separate commit. It's totally OK by me to fix this kind of stuff in an unrelated PR, but if you keep the commit separate, it's easier to track what's going on in the commit history and if you have to roll something back, you don't have 2 unrelated things in a single commit that you might have to rollback together, when you really only want to rollback one of them.

File: test/projectOverviewResponse.test.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('Project Overview Response', () => {
     ];
     const expectedResult = smb()
       .attachment()
-      .text("*  *facebook* facebook.com\n*  *google* google.com")
+      .text('*  *facebook* facebook.com\n*  *google* google.com')
```
Comment (nrambeck) [inline test/projectOverviewResponse.test.js]: Same as the other comment. Try to keep these linting fixes in a separate commit. For example, the 2 linting fixes could go in a commit with a message of: 'fix various linting errors"

File: bot.js [hunk 3]
```diff
@@ -71,6 +71,10 @@ var bot_options = {
     studio_command_uri: process.env.studio_command_uri
 };
 
+const SQLDriver = require('./lib/dbDriver/mysql');
+
+global.projectBot = {};
+projectBot.driver = new SQLDriver();
```
Comment (nrambeck) [inline bot.js]: I'd recommending changing the name of this property to `projectBot.dbDriver` to make it abundantly clear what it is. Drivers are not just for databases, so making a more descriptive name can help with clarity.

File: lib/actions/projectOverviewResponse.js (hunk 1 above)
Comment (colbyallenc) [reply lib/actions/projectOverviewResponse.js]: got it. I have to keep the consistency going and not be lazy on my commits.

File: bot.js (hunk 3 above)
Comment (colbyallenc) [reply bot.js]: makes sense!
